content/c/concepts/operators/terms/sizeof/sizeof.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'sizeof()'
+Description: 'Returns an integer representing the memory size of a data type or variable in bytes.'
```

Comment:
We should keep the purpose of the `sizeof()` function first, followed by the return type.

```suggestion
Description: 'Determines the memory size (in bytes) of a given data type or variable and returns it as an integer.'
```